### PR TITLE
Permission changes

### PR DESCRIPTION
**Pull request by PotatoPresident**
changed permissions to use sub permissions

### CODE REVIEW

File: src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java [hunk 1]
```diff
@@ -41,23 +41,23 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                     .then(literal("list")
                         .executes(AliasCommand::listAliases))
                     .then(literal("add")
-                        .requires(x -> Permissions.check(x, "pingspam.addownalias", 0))
+                        .requires(x -> Permissions.check(x, "pingspam.alias.own.add", true))
```
Comment (BasiqueEvangelist) [inline src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java:44]: The permission in the README is `pingspam.alias.addown`, but here it's `pingspam.alias.own.add`.

File: src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java [hunk 2]
```diff
@@ -41,23 +41,23 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
...
                         .then(argument("alias", StringArgumentType.word())
                             .executes(AliasCommand::addAliases)))
                     .then(literal("remove")
-                        .requires(x -> Permissions.check(x, "pingspam.removeownalias", 0))
+                        .requires(x -> Permissions.check(x, "pingspam.alias.own.remove", true))
```
Comment (BasiqueEvangelist) [inline src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java:48]: See above.

File: src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java [hunk 3]
```diff
@@ -41,23 +41,23 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
...
                         .then(argument("alias", StringArgumentType.word())
                             .executes(AliasCommand::removeAlias)))
                     .then(literal("player")
                         .then(argument("player", EntityArgumentType.player())
                             .then(literal("list")
                                 .executes(AliasCommand::listPlayerAliases))
                             .then(literal("add")
-                                .requires(x -> Permissions.check(x, "pingspam.addplayeralias", 2))
+                                .requires(x -> Permissions.check(x, "pingspam.alias.player.add", 2))
```
Comment (BasiqueEvangelist) [inline src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java:56]: See above.

File: src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java [hunk 4]
```diff
@@ -41,23 +41,23 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
...
                                 .then(argument("alias", StringArgumentType.word())
                                     .executes(AliasCommand::addPlayerAlias)))
                             .then(literal("remove")
-                                .requires(x -> Permissions.check(x, "pingspam.removeplayeralias", 2))
+                                .requires(x -> Permissions.check(x, "pingspam.alias.player.remove", 2))
```
Comment (BasiqueEvangelist) [inline src/main/java/me/basiqueevangelist/pingspam/commands/AliasCommand.java:60]: See above.

**Review by BasiqueEvangelist — CHANGES_REQUESTED** [review]

**Comment by Jetsparrow** [comment]
@BasiqueEvangelist That's the point of the change, to allow granting pingspam.alias.own.*

**Comment by BasiqueEvangelist** [comment]
> @BasiqueEvangelist That's the point of the change, to allow granting pingspam.alias.own.*

I know. The problem is the README was also changed, but the permissions in it don't match the permissions used in code.

**Review by BasiqueEvangelist — APPROVED** [review]

**Pull request merged**